index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
Please remove all the JavaScript code from your HTML file.

Reply:
i have done that, my code now looks clean, thanks